components/raftstore/src/store/msg.rs

```diff
@@ -847,6 +847,10 @@ where
     },
 
     GcSnapshotFinish,
+
+    AwakenRegions {
+        abnormal_stores: Vec<u64>,
```

Comment:
Could we skip all `store ids` which are not equals to `self store id`?

Reply:
Nope, as mentioned above, we wanna push down the filtration from PD to TiKV side by `stores`. And without `abnormal stores`, we won't know which store is abnormal.

Reply:
Yes you'r right